checkov/terraform/checks/resource/aws/S3MFADelete.py

```diff
@@ -17,10 +17,11 @@ def scan_resource_conf(self, conf):
         if 'versioning' in conf.keys():
             versioning_block = conf['versioning']
             for block in versioning_block:
-                if 'mfa_delete' in block.keys():
-                    if block['mfa_delete']:
+                if type(block) is dict:
```

Comment:
@arielkru Here too - if it is not a dict, we won't be able to assess it so I think it should pass